### ISSUE TITLE
Labelling or coding?

### ISSUE THREAD

**Issue by benel** [Hypertopic/Cassandre#115]
For very good reasons, the handbook states that it is better to name the analysis action "labelling" (in French : "étiqueter") instead of "coding" (which is widely used but refers to a time where "codebooks" had to be defined before the analysis to map an ID with a string).

In the software interface, the icon is clearly a label, but the texts still use "[coding memo](https://github.com/Hypertopic/Cassandre/search?l=JSON&q=coding+memo)".

**Comment by christophe-lejeune** [comment]
This difference between French and English intended. 

But the situation is even trickiest that what you describe. 

In the _handbook of current developments in GTM_, Udo Kelle (2019) indeed explains that, at the very beginning, "coding" was used to refer to codebooks in content analysis (a quantitative technique, with strict own rules, alien to Cassandre and what we try to do here). Later, the term was coined in qualitative research, namely by Barney Glaser, under the direction of Paul Lazarsfeld (who promoted public opinion surveys, quantitative methods and, logically, content analysis). In 1960, Blanche Geer and Howard Becker adopted "coding" in the classical study (_Boys in white_) that Anselm Strauss participated. It is thus no surprise that the term "coding" still appears as a key technique when Barney Glaser & Anselm Strauss formalized qualitative research/grounded theory method.

However, "coding" refers to different practice while used to Lasswell & Lazarsfeld, Geer & Becker or Glaser & Strauss. 

To make a clear distinction, I proposed to translate differently the various occurrences of "coding" in the English literature. 
* Micro-analysis coding is translated as "annoter"
* Methodological coding is translated as "étiqueter"
* Operational coding (sometimes criticized as "coding trap") is translated as "indexation thématique"
* Coding memo (the memo that accompanies the coding practice) is translated as "compte-rendu de codage". I decided to keep "codage" here, because coding memos are used even without methodological codes. Any memos conceptualizing from (and grounded in) in a transcription/field memo is typically a coding memo. 
* Phrases such as initial coding, open coding, axial coding and selective coding are translated as "codage" ("ouvert, axial et sélectif"). 

The reason why I choose "étiqueter" (label) for (conceptual) coding is twofold : 
* The founding book by Glaser & Strauss originally defines coding with a reference to "label" ("étiquette")
* In another (every day) field, labeling refers to a phenomena with common features : according to Howard Becker (in _Outsiders_), a label informs to what is labeled (here, the empirical material) and who is labeling (the analyst). 

While writing the first version of the manual, this choice appeared unproblematic. However, today, the situation in made more complex, since Linda Liska Belgrave and Kapriskie Seide elaborated on Kathy Charmaz's manual that "While codes themselves might be labels, coding is more than labeling" (in "Coding for Grounded Theory", a chapter of the _Handbook of current developments in GTM_).

Given this recent development, "coding is not labeling". 
* **Coding** is the "right" way to analyze, it is necessarily accompanied with memoing that together help conceptualize. 
* **Labeling** is considered as a trap, leading to mere description.

In conclusion, the French "étiquetage" should thus be translated as English "coding".

**Comment by benel** [comment]
OK. I understand that terms have different connotations depending on language, domains and practice... Moreover scientific terms are often arbitrary conventions. 

I cannot help thinking that"labelling" is a more open and physical metaphor than "coding". But I understand that Cassandre must comply with GTM literature (at least in its English version). 

Because I use the French version it will be OK for me. But I will keep the secret hope that the GTM community will discuss one day the hidden connotations of the terms they choose to use (and the consequences on learners and early practitioners  that only have a very shallow understanding of what GTM is).
